packages/ember/tests/reexports_test.js

```diff
@@ -108,7 +108,6 @@ QUnit.module('ember reexports');
   ['removeObserver', 'ember-metal'],
   ['_suspendObserver', 'ember-metal'],
   ['_suspendObservers', 'ember-metal'],
-  ['required', 'ember-metal'],
```

Comment:
I believe we are going to have to guard this

Reply:
ok